.github/workflows/ci.yml

```diff
@@ -36,5 +36,11 @@ jobs:
             cat errors.csv
             exit 1
           fi
+        - name: Validate html including missing uuid
+          run: |
+            validate-mbz-html html errors.csv html --uuids-populated
+            CSV_LINES=$(wc -l < errors.csv )
+            echo "Validation errors including missing uuid errors found:"
```

Comment:
Since we're going to print this even if there are no missing UUID errors, maybe we can do:

```bash
echo "$(($CSV_LINES-1)) missing UUID errors found in html"
```